README.md

```diff
@@ -527,38 +570,137 @@ respect cancellation, the shutdown may never complete.
 The Python SDK is built to work with Python 3.7 and newer. It is built using
 [SDK Core](https://github.com/temporalio/sdk-core/) which is written in Rust.
 
-### Local development environment
+### Building
+
+#### Prepare
+
+To build the SDK from source for use as a dependency, the following prerequisites are required:
+
+* [Python](https://www.python.org/) >= 3.7
+* [Rust](https://www.rust-lang.org/)
+* [pipx](https://github.com/pypa/pipx#install-pipx) (only needed for installing the two dependencies below)
+* [poetry](https://github.com/python-poetry/poetry) `pipx install poetry`
+* [poe](https://github.com/nat-n/poethepoet) `pipx install poethepoet`
+
+With the prerequisites installed, first clone the SDK repository recursively:
+
+```bash
+git clone --recursive https://github.com/temporalio/sdk-python.git
+cd sdk-python
+```
+
+Use `poetry` to install the dependencies with `--no-root` to not install this package:
```

Comment:
Might want to briefly explain why `--no-root`

Reply:
"to not install this package". But I can provide more detail.

Reply:
Done